apps/studio/components/grid/components/header/Header.tsx

```diff
@@ -117,7 +116,6 @@ const DefaultHeader = ({
   return (
     <div className="flex items-center gap-4">
       <div className="flex items-center gap-2">
```

Comment:
Should be using correct markup here. Like role="menu"

Can deal with in a follow up PR

Reply:
https://www.w3.org/WAI/tutorials/menus/application-menus/

Probably this is the right implantation for this. Although this is a bit of a guess

Reply:
Worth mentioning the <Trigger> components for things like Popover already have aria labels for things like "haspopup", but you'll need to probably set role="menuitem" on them